packages/exo/src/exo-makers.js

```diff
@@ -228,6 +254,21 @@ export const defineExoClassKit = (
     receiveRevoker(revoke);
   }
 
+  if (receiveAmplifier) {
+    const amplify = (aFacet, facetName) => {
+      for (const contextMap of values(contextMapKit)) {
```

Comment:
Can this `amplify` function be considered approximately O(1) for the purpose of [limiting work by message size](https://github.com/Agoric/agoric-sdk/wiki/Limit-work-by-message-size)?

If not, please document that LOUDLY. Is `contextMapKit` typically high-cardinality? What causes it to grow?

In [the crabble review](https://github.com/Agoric/agoric-sdk/issues/8465), we looked for loops and `.map()` and such, but we didn't consider the possibility that functions in the Agoric SDK (including endo) might have running time that grows much faster than their arguments.

Reply:
> Can this `amplify` function be considered approximately O(1) for the purpose of [limiting work by message size](https://github.com/Agoric/agoric-sdk/wiki/Limit-work-by-message-size)?

Yes. We assume that the number of facet names of a class kit is very small, so this O(n) is of a tiny n. Further, fr any given class kit it cannot grow dynamically. Resolving.